app/src/main/java/com/bignerdranch/android/geoquiz/QuizViewModel.kt

```diff
@@ -5,6 +5,7 @@ import androidx.lifecycle.ViewModel
 class QuizViewModel : ViewModel() {
 
     var currentIndex = 0
+    var score = 0.0
```

Comment:
this goes to this variable and the others in this class:

if you are exposing them to outside, its better to expose a final `val` that cannot be mutated, which internally (from here) references a mutable variable, i.e:
```
private var scoreData = 0.0
val score get() = scoreData
```
that way you are ensuring that you can _only_ read that property from the outside, but do not have the direct access to its state. This comes in handy down the line, when your code gains on complexity

Reply:
done